packages/core/upload/shared/contracts/files.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * Used to store the folder structure inside the media library.
+ */
+import type { Data, Modules } from '@strapi/types';
+import { errors } from '@strapi/utils';
+
+type SortOrder = 'ASC' | 'DESC';
+
+type SortKey = 'createdAt' | 'name';
+
+/**
+ * GET /upload/files - Get files
+ */
+export interface File {
```

Comment:
A bit different, cause this is what we use to store it on db, but I saw some differences between this contract and the db definition
https://github.com/strapi/strapi/blob/bc9f526e2ba2b0b11ae11df6628346dc58871e98/packages/providers/upload-local/src/index.ts#L10

There are some fields that could be optional here (width, height, caption, alternativeText, ext)